primitives/src/constants.rs

```diff
@@ -181,6 +181,6 @@ pub const TECHNICAL_DID: IdentityId = IdentityId(*did::TECHNICAL_COMMITTEE_DID);
 pub const UPGRADE_DID: IdentityId = IdentityId(*did::UPGRADE_COMMITTEE_DID);
 
 /// Prefixes for scheduled actions
-pub const SETTLEMENT_INSTRUCTION_EXECUTION: [u8; 27] = *b"SETTLEMENT_INSTRUCTION_EXEC";
+pub const SETTLEMENT_INSTRUCTION_EXECUTION: [u8; 20] = *b"SETTLEMENT_INST_EXEC";
```

Comment:
We should check how this will impact settlements that are scheduled before the v8 upgrade.

Reply:
If the Substrate scheduler has lowered the length of named tasks, it might have a migration to truncate the task names.